index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/5706a27a8e.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <section id="main">
+        <section class="statusBar bcGrey">
+            <div class="statusBarContent">
+                <div class="time"><img src="img/Time.png" alt=""></div>
+                <div class="statusIcons">
+                    <img src="img/Cellular.png" alt="">
+                    <img src="img/Wifi.png" alt="">
+                    <img src="img/Battery.png" alt="">
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Great job with your design 👏 . Although, this part of the design shouldn't be added to your design. Please remove it 🙏 

![image](https://user-images.githubusercontent.com/74814780/205103474-0af7a466-640f-42ba-91b9-333070e1ff23.png)

Reply:
Fixed.